### PR TITLE
reorganized masthead css

### PR DESCRIPTION
**Pull request by mirabelrice**

### CODE REVIEW

File: app/assets/stylesheets/regions/mastheads/boxes-masthead.scss [hunk 1]
```diff
@@ -227,47 +227,4 @@
     min-height: 340px !important;
     padding-bottom: 0;
   }
-}
-
```
Comment (mirabelrice) [inline app/assets/stylesheets/regions/mastheads/boxes-masthead.scss]: Note: I deleted `app/assets/stylesheets/widgets/masthead.scss` and created `app/assets/stylesheets/widgets/boxes-masthead.scss`. I'm not sure why git is making it seem like i just moved the file.

File: app/assets/stylesheets/regions/mastheads/slider-masthead_old.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
+.rounded-slider {
```
Comment (tatwell) [inline app/assets/stylesheets/regions/mastheads/slider-masthead_old.scss]: Why not `slider-masthead-old.scss`?

File: app/assets/stylesheets/regions/mastheads/boxes-masthead.scss (hunk 1 above)
Comment (tatwell) [inline app/assets/stylesheets/regions/mastheads/boxes-masthead.scss]: Looks like it's all the same content. Was all the code in there boxes-related?

File: app/assets/stylesheets/regions/mastheads/slider-masthead_old.scss (hunk 2 above)
Comment (mirabelrice) [inline app/assets/stylesheets/regions/mastheads/slider-masthead_old.scss]: Yeah I can change that, i just thought it differentiated it more

File: app/assets/stylesheets/regions/mastheads/boxes-masthead.scss (hunk 1 above)
Comment (mirabelrice) [inline app/assets/stylesheets/regions/mastheads/boxes-masthead.scss]: No, the code that is showing as deleted was moved `app/assets/stylesheets/regions/slider-masthead-old.scss`

Comment (tatwell) [inline app/assets/stylesheets/regions/mastheads/boxes-masthead.scss]: So all the remaining code is boxes-related?

Comment (mirabelrice) [inline app/assets/stylesheets/regions/mastheads/boxes-masthead.scss]: Well yes the remaining code in this file (boxes-masthead.scss) is for boxes. Some of the code that was originally in the masthead.scss was moved to slider-masthead-old.scss though.

Comment (mirabelrice) [inline app/assets/stylesheets/regions/mastheads/boxes-masthead.scss]: I just wanted to make it clear that I deleted `masthead.scss`.

Comment (tatwell) [inline app/assets/stylesheets/regions/mastheads/boxes-masthead.scss]: Thanks for clarification. Looks good to me. 👍

**Pull request merged**